src/components/ec-filter-popover/ec-filter-popover.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    class="ec-filter-popover"
+    data-test="ec-filter-popover"
+  >
+    <ec-popover
+      popover-class="ec-popover"
```

Comment:
Why do you have to re-declare this? I think the default value is already set to ec-popover.

Reply:
In the snapshots was `popover-class=""` thought I needed to re-declare it

Reply:
That might have been generated before you used shallowMount(). It doesn't make sense to re-declare this, look what it generates in the browser:

![Screenshot from 2020-10-20 13-07-40](https://user-images.githubusercontent.com/5777410/96583971-71b5b580-12d5-11eb-856a-054704d378dc.png)

ec-popover is there twice. Just remove this and update snapshots.